SmartDeviceLink/private/SDLStreamingVideoLifecycleManager.m

```diff
@@ -856,6 +1167,18 @@ - (BOOL)isStreamingSupported {
     return (self.systemCapabilityManager != nil) ? [self.systemCapabilityManager isCapabilitySupported:SDLSystemCapabilityTypeVideoStreaming] : YES;
 }
 
+- (void)setPreferredResolutions:(NSArray<SDLImageResolution *> *)resolutions {
+    _preferredResolutions = resolutions;
+    _preferredResolutionIndex = 0;
+    [self didChangeValueForKey:@"preferredResolutions"];
+}
+
+- (void)setPreferredFormats:(NSArray<SDLVideoStreamingFormat *> *)formats {
+    _preferredFormats = formats;
+    _preferredFormatIndex = 0;
+    [self didChangeValueForKey:@"preferredFormats"];
+}
```

Comment:
Are these setters needed? There doesn't appear to be any place in the code where `preferredResolutions` or `preferredFormats` is observed.